App.js

```diff
@@ -26,68 +28,66 @@ export default class App extends React.Component {
       Roboto_medium: require("native-base/Fonts/Roboto_medium.ttf"),
       ...Ionicons.font
     });
-    this.setState({ isReady: true });
-    this.setState({ screen: "map" });
+
+    this.setState({ isReady: true, screen: "map" });
   }
 
   load_user = () => {
-    if (this.state.searchedText.length > 0) {
+    const { searchedText } = this.state;
+
+    if (searchedText.length > 0) {
       this.setState({ isLoading: true });
-      catchUserTLFromId(this.state.searchedText).then(datas => {
+      getDataTLFromId(this.state.searchedText).then(datas => {
         this.setState({ user_tl: datas, isLoading: false });
       });
     }
   };
-  _searchTextInputChanged = text => {
```

Comment:
Usually, `_` is used for meaning a variable is declared but not used, that's why I removed it